logic/auth.js

```diff
@@ -32,7 +32,7 @@ async function sleepSeconds(seconds) {
 
 // Caches the password.
 function cachePassword(password) {
-    devicePassword = password;
+    devicePassword = Buffer.from(password, "base64").toString("utf-8");
```

Comment:
Can we do the decoding once as soon as the route is hit instead of everywhere it's used in the logic modules?

Reply:
I tried yesterday and didn't succeed (maybe I've missed something)